AppCenterDataStorage/AppCenterDataStorage/Internal/LocalStore/MSDBDocumentStore.m

```diff
@@ -0,0 +1,79 @@
+    NSDictionary *columnIndexes = [MSDBStorage columnsIndexes:schema];
+    _idColumnIndex = ((NSNumber *)columnIndexes[kMSAppDocumentTableName][kMSIdColumnName]).unsignedIntegerValue;
+    _partitionColumnIndex = ((NSNumber *)columnIndexes[kMSAppDocumentTableName][kMSPartitionColumnName]).unsignedIntegerValue;
+    _documentIdColumnIndex = ((NSNumber *)columnIndexes[kMSAppDocumentTableName][kMSDocumentIdColumnName]).unsignedIntegerValue;
+    _documentColumnIndex = ((NSNumber *)columnIndexes[kMSAppDocumentTableName][kMSDocumentColumnName]).unsignedIntegerValue;
+    _eTagColumnIndex = ((NSNumber *)columnIndexes[kMSAppDocumentTableName][kMSETagColumnName]).unsignedIntegerValue;
+    _expirationTimeColumnIndex = ((NSNumber *)columnIndexes[kMSAppDocumentTableName][kMSExpirationTimeColumnName]).unsignedIntegerValue;
+    _downloadTimeColumnIndex = ((NSNumber *)columnIndexes[kMSAppDocumentTableName][kMSDownloadTimeColumnName]).unsignedIntegerValue;
+    _operationTimeColumnIndex = ((NSNumber *)columnIndexes[kMSAppDocumentTableName][kMSOperationTimeColumnName]).unsignedIntegerValue;
+    _pendingOperationColumnIndex = ((NSNumber *)columnIndexes[kMSAppDocumentTableName][kMSPendingDownloadColumnName]).unsignedIntegerValue;
+  }
+  return self;
+}
+
+#pragma mark - Table Management
+
+//TODO work item created to tracl this implementation
+- (BOOL)createWithPartition:(NSString *)__unused partition
+                   document:(MSDocumentWrapper *)__unused document
+               writeOptions:(MSWriteOptions *)__unused writeOptions {
+  return YES;
+}
+
+- (NSUInteger) createUserStorageWithAccountId:(NSString *)accountId {
+    MSDBSchema *schema = @{[NSString stringWithFormat:kMSUserDocumentTableNameFormat, accountId] : [MSDBDocumentStore tableSchema]};
+  
+    // Create table based on the schema.
+    return (int)[self.dbStorage createTablesWithSchema:schema];
+}
+
+- (BOOL)deleteUserStorageWithAccountId:(NSString *)accountId {
+  NSString *tableName = [NSString stringWithFormat:kMSUserDocumentTableNameFormat, accountId];
+  return [self.dbStorage dropTable:tableName];
+}
+
++ (NSArray<NSDictionary<NSString *, NSArray<NSString *> *> *> *)tableSchema {
+
+  // TODO create composite key for partition and the document id
+  return @[
+    @{kMSIdColumnName : @[ kMSSQLiteTypeInteger, kMSSQLiteConstraintPrimaryKey, kMSSQLiteConstraintAutoincrement ]},
+    @{kMSPartitionColumnName : @[ kMSSQLiteTypeText, kMSSQLiteConstraintNotNull ]},
+    @{kMSDocumentIdColumnName : @[ kMSSQLiteTypeText, kMSSQLiteConstraintNotNull ]}, @{kMSDocumentColumnName : @[ kMSSQLiteTypeText ]},
+    @{kMSETagColumnName : @[ kMSSQLiteTypeText ]}, @{kMSExpirationTimeColumnName : @[ kMSSQLiteTypeInteger ]},
+    @{kMSDownloadTimeColumnName : @[ kMSSQLiteTypeInteger ]}, @{kMSOperationTimeColumnName : @[ kMSSQLiteTypeInteger ]},
+    @{kMSPendingDownloadColumnName : @[ kMSSQLiteTypeText ]}
```

Comment:
Operation downloadtime is missing. 
https://msmobilecenter.visualstudio.com/Mobile-Center/_wiki/wikis/Mobile-Center.wiki?wikiVersion=GBwikiMaster&pagePath=%2FHome%2FTeams%2FMBaaS%2FStorage%2FSpecifications%2FLocal%20Documents%20Storage&pageId=2128

Reply:
It is there. The indentation is missing